doc/nrf/ug_wifi.rst

```diff
@@ -0,0 +1,141 @@
+.. _ug_wifi:
+
+Wi-Fi
+#####
+
+.. contents::
+   :local:
+   :depth: 2
+
+Wi-Fi is the branding used by the Wi-Fi Alliance to describe the family of wireless local area network standards specified by the IEEE802.11 Working Group.
+The two industry bodies work closely together, with the Wi-Fi Alliance focusing on the requirements and certification of the technology, and the IEEE focusing on the underlying technical specifications.
+The Wi-Fi Alliance is responsible for the Wi-Fi CERTIFIED accreditation program, which aims to achieve high level of interoperability, reliability, and security compliance.
```

Comment:
```suggestion
The Wi-Fi Alliance is responsible for the Wi-Fi CERTIFIED accreditation program, which aims to achieve high level of interoperability, reliability, and security compliance. Further information related to certification on the nRF70 Series can be found here [link to infocenter WFA certification doc].
```

Reply:
Infocenter docs are not available yet.

Reply:
The infocenter structure for guidelines is still being worked out. This will not be known until Wed/Thurs this week.